libs/ui/src/pure/ClosableBanner/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { atomWithStorage } from 'jotai/utils'
+import { getJotaiMergerStorage } from '@cowprotocol/core'
+import { useAtom } from 'jotai'
+import { useCallback } from 'react'
+
+// true - when banner closed
+type ClosableBannersState = Record<string, true | undefined>
+
+interface ClosableBannerCallback {
+  (close: () => void): JSX.Element
+}
+
+export interface ClosableBannerProps {
+  storageKey: string
+  callback: ClosableBannerCallback
+}
+
+const DEFAULT_STATE: ClosableBannersState = {}
+
+const closableBannersStateAtom = atomWithStorage<ClosableBannersState>(
+  'closable-banners',
```

Comment:
Should this be versioned as well? and camelCase to follow the pattern of other keys?

```suggestion
  'closableBanners:v0',
```

Reply:
Fixed